src/content/docs/en/reference/api-reference.mdx

```diff
@@ -52,14 +52,19 @@ Markdown files have the following interface:
 export interface MarkdownInstance<T extends Record<string, any>> {
   /* Any data specified in this file's YAML frontmatter */
 	frontmatter: T;
-  /* The file path of this file */
+  /* The absolute file path of this file */
 	file: string;
   /* The rendered path of this file */
 	url: string | undefined;
   /* Astro Component that renders the contents of this file */
 	Content: AstroComponent;
+  /** Raw Markdown file content, excluding layout HTML and YAML frontmatter */
+	rawContent(): string;
+  /** Markdown file compiled to HTML, excluding layout HTML */
+	compiledContent(): string;
```

Comment:
I believe these are only available for Markdown but **not** MDX. Do you think that should be mentioned here?

Similar to in the Markdown docs: https://docs.astro.build/en/guides/markdown-content/#exported-properties

Reply:
Indeed, they are only available for Markdown. I had hesitated to add them but since this is `MarkdownInstance` and not `MDXInstance` I figured they should appear. But a `Markdown only` comment makes perfect sense!